core/src/components/input/input.ionic.scss

```
@@ -35,6 +35,18 @@
   min-height: 56px;
 }
 
+// Ionic Input Password Toggle Sizes
+// --------------------------------------------------
+
+:host ion-input-password-toggle {
+  --size: 40px;
+  --icon-size: #{globals.dynamic-font(16px)};
+}
+
+:host(.input-size-large) ion-input-password-toggle {
+  --size: 50px;
```

```suggestion
  --size: #{globals.$ionic-scale-1200};
```

I spoke with Beth and this should be `48px` for `large` which is the above scale token